src/index.html

```diff
@@ -39,7 +39,7 @@
 
 <noscript>
   <!-- If user's javascript is disabled -->
```

Comment:
Have you considered removing this comment?

Reply:
Done